gnome-initial-setup/pages/language/gis-language-page.c

```diff
@@ -42,8 +42,6 @@
 
 struct _GisLanguagePagePrivate
 {
-  GtkWidget *logo;
```

Comment:
Is this code actually harmful? Otherwise I prefer to avoid carrying another downstream patch.